src/Microsoft.ML.DnnAnalyzer/Microsoft.ML.DnnAnalyzer/DnnAnalyzer.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.Internal.Utilities;
+using Microsoft.ML.Transforms.TensorFlow;
+using System.Linq;
+
+namespace Microsoft.ML.DnnAnalyzer
+{
+    public static class DnnAnalyzer
+    {
+        public static void Main(string[] args)
+        {
+            using (var env = new TlcEnvironment())
```

Comment:
>TlcEnvironment [](start = 33, length = 14)

isn't it feel weird to create environment only to write something to console?
Why can't you just do Console.Writeline?